src/Foundation/Exception/Handler.php

```diff
@@ -176,14 +176,14 @@ protected function callCustomHandlers($throwable, $fromConsole = false)
             // at least some errors, and avoid errors with no data or not log writes.
             try {
                 $response = $handler($throwable, $code, $fromConsole);
+            } catch (Throwable $t) {
+                $response = $this->convertExceptionToResponse($t);
             }
-            catch (Throwable $t) {
-                $response = $this->convertThrowableToResponse($t);
-            }
+
             // If this handler returns a "non-null" response, we will return it so it will
             // get sent back to the browsers. Once the handler returns a valid response
             // we will cease iterating through them and calling these other handlers.
-            if (isset($response) && ! is_null($response)) {
+            if (isset($response)) {
```

Comment:
Why was this required?

Reply:
`isset` and `!is_null` is the same test.